clang/lib/AST/ASTStructuralEquivalence.cpp

```diff
@@ -1496,9 +1615,37 @@ static bool IsStructurallyEquivalent(StructuralEquivalenceContext &Context,
     return false;
   }
 
-  if (Field1->isBitField())
-    return IsStructurallyEquivalent(Context, Field1->getBitWidth(),
-                                    Field2->getBitWidth());
+  if ((Field1->isBitField() || Field2->isBitField()) &&
```

Comment:
Was this a previous bug?

Reply:
Yes, it was a previous bug and I am testing it in the n3037.c test by testing in both orders.